boot.sh

```diff
@@ -31,6 +31,7 @@ readonly DOCKER_CLIENT_LEGACY="/usr/bin/docker"
 readonly DOCKER_CLIENT_LATEST="/opt/docker/docker"
 readonly BOOT_WAIT_TIME=10
 readonly SWAP_FILE_PATH="/root/.__sh_swap__"
+readonly DOCKER_VERSION=$(docker version --format '{{.Server.Version}}')
```

Comment:
lets not do this here. keep the env as empty string here and then calculate it later as one of the steps